lib/common.js

```diff
@@ -798,7 +799,8 @@ function parseArgs(args) {
   var opts = {};
   for (var i = 0; i < args.length; i++) {
     var arg = args[i];
-    if (arg.indexOf('=') === -1) throw new Error('Invalid argument format: ' + arg);
+    if (arg.indexOf('=') === -1)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)